installinator-common/src/progress.rs

```diff
@@ -262,17 +263,28 @@ pub enum WriteStepId {
 
 /// The error that occurred.
 #[derive(Debug, Error)]
-#[error(
-    "writing {component} to slot {slot} failed \
+pub enum WriteError {
+    #[error(
+        "writing {component} to slot {slot} failed \
      after {written_bytes}/{total_bytes} bytes"
```

Comment:
Indent needs to be matched with previous line here.

Reply:
:+1: fixed in 2a518741